lib/mail_room/connection.rb

```diff
@@ -50,6 +50,7 @@ def wait
 
         process_mailbox
       rescue Net::IMAP::Error, IOError
+        MailRoom.structured_logger.warn("#{@mailbox.context} Disconnected. Resetting...")
```

Comment:
Maybe we want to put `@mailbox.context` in a separate JSON field rather than just being part of `message`?

Reply:
I like that idea - going for it!~